1/README.md

```diff
@@ -21,16 +22,36 @@ This Specification is a [free and open standard](http://www.digistan.org/open-st
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
 
 ## Tango Specification
-The TANGO control system is a device-oriented controls toolkit for controlling any kind of hardware or software and building SCADA systems.
-
+The TANGO control system is a toolkit for building distributed control systems which are device oriented. 
 
 ### Goals
 
-Tango aims to 
+Tango aims to be extremely flexible and can be used to interface any kind of hardware or software system which can be accessed via one of the software languages supported by Tango (C++, Python, Java and JavaScript).
+
+The goal of Tango is to provide a way of representing all physical and software systems which need to be controlled by a control system as Tango Devices.
+
+Devices are software components which have Commands (methods), Attributes (data) and Properties (configuration) and are accessible on the network or locally.
+
+A Tango control system consists of a collection of Tango Devices, an (optional) Database and a set of applications for configuring and controlling the Devices via well defined Application Programmers Interfaces (APIs). Tango defines and maintains official APIs for C++, Python, Java, JavaScript and REST.
+
+The Tango Database serves the purpose of defining and declaring all Devices in the control system and their configuration parameters. The Database also stores the network address of each Device and serves as Naming Service for clients who need to establish a connection to any or all Devices.
 
 The software used in large facilities can be compared with the software for stock markets – huge amount of data must be displayed on the monitor in a real time and being processed and being saved in databases for post processing.
 
-So, the toolkit was mainly developed for research facilities needs, but the idea and concept (philosophy) behind was to create a framework.
+The Tango toolkit was mainly developed for research facilities needs, but the idea and concept (philosophy) behind was to create a framework for control systems in general. The concept is based on the idea that the concept of an object or component is ideally suited to designing and implementing control systems. TANGO was based on this concept and can be described as an object oriented control system. A number of concepts were defined as being necessary to implement a TANGO control system (ref "TANGO - AN OBJECT ORIENTED CONTROL SYSTEM BASED ON CORBA" by Chaize et. al., ICALEPCS 1999). These are:
```

Comment:
I'd rephrase this as:
"The Tango toolkit was mainly developed for research facilities needs, but the idea and concept (philosophy) behind was to create a framework for control systems in general. It is based on the idea that distributed objects are really suited to design and implement control systems. TANGO was based on this concept, and can be described as an Object Oriented distributed control system. A number of requirements were defined in the original design (ref... TANGO - An Object ...):

1. ...
2. ...